README.md

```diff
@@ -10,7 +10,10 @@ annotations.
 - Enable workflows in the GitHub repository
 - Under [secrets](https://github.com/Sage-Bionetworks/amp-ad-metadata-dictionary/settings/secrets/actions) click 'New repository secret'
 - Enter secrets for `RSCONNECT_USER`, `RSCONNECT_TOKEN`, and `RSCONNECT_SECRET`, the values for which are saved in Sage's LastPass.
-- Trigger the GitHub action.
+- Push to new branch on Github
+- Create pull request with at least one reviewer to master branch
+- Upon approval of pull request, the app will deploy to staging.
 - Check out the app here: https://sagebio.shinyapps.io/amp-ad-metadata-dictionary-staging.
-- After verifying correctness, create a Git branch named release*, e.g., `release-1.0`.
+- After verifying correctness, create pull request with at least one reviewer to prod branch
```

Comment:
I'd suggest updating this to be more explicit:
After verifying correctness, create a pull request to merge master to prod; at least one reviewer must approve this PR before it can be merged to trigger the production deployment

Reply:
Thanks, clarity is good.